ViewLayer/DebugDisplay/MotorFaultPage/MotorFaultView/MotorFaultView.cpp

```diff
@@ -0,0 +1,2819 @@
+    , dclReducedDueToTemperature_ (new QLabel())
+    , labelBcount_ (0)
+{
+
+    // Setting up Vertical bar (wasn't letting me setup in namespace)
+    QScrollBar* verticalBar = new QScrollBar;
+    verticalBar->setStyleSheet(QString::fromUtf8("QScrollBar:vertical {"
+                               "    background:rgba(83, 83, 84);"
+                               "    width:10px;    "
+                               "    margin: 0px 0px 0px 0px;"
+                               "}"
+                               "QScrollBar::handle:vertical {"
+                               "    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,"
+                               "    stop: 0 rgb(255, 192, 33), stop: 0.5 rgb(255, 192, 33), stop:1 rgb(255, 192, 33));"
+                               "    min-height: 0px;"
+                               "    border-radius: 5px;"
+                               "}"
+                               "QScrollBar::add-line:vertical {"
+                               "    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,"
+                               "    stop: 0 rgb(255, 192, 33), stop: 0.5 rgb(255, 192, 33),  stop:1 rgb(255, 192, 33));"
+                               "    height: 0px;"
+                               "    subcontrol-position: bottom;"
+                               "    subcontrol-origin: margin;"
+                               "}"
+                               "QScrollBar::sub-line:vertical {"
+                               "    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,"
+                               "    stop: 0  rgb(255, 192, 33), stop: 0.5 rgb(255, 192, 33),  stop:1 rgb(255, 192, 33));"
+                               "    height: 0 px;"
+                               "    subcontrol-position: top;"
+                               "    subcontrol-origin: margin;"
+                               "}"
+                                                ));
+
+    ui_.motor0ScrollArea().setVerticalScrollBar(verticalBar);
+    ui_.motor1ScrollArea().setVerticalScrollBar(verticalBar);
+    ui_.batteryScrollArea().setVerticalScrollBar(verticalBar);
+
+    QLayout* layout0 = ui_.motor0ContentsWidget().layout();
+    int current_width0 = ui_.motor0ContentsWidget().size().width();
+
+    QLayout* layout1 = ui_.motor1ContentsWidget().layout();
+    int current_width1 = ui_.motor1ContentsWidget().size().width();
+
+    QLayout* layoutB = ui_.batteryContentsWidget().layout();
+    int current_widthB = ui_.batteryContentsWidget().size().width();
+
+    // Motor 0
+    badMotorPositionHallSequence0Fault_.resize(current_width0, height );
+    badMotorPositionHallSequence0Fault_.setText("Bad Motor Position Hall Sequence Error");
+    badMotorPositionHallSequence0Fault_.setStyleSheet(errorStyleSheet);
+    badMotorPositionHallSequence0Fault_.setFixedSize(current_width0, height);
+    layout0->addWidget(&badMotorPositionHallSequence0Fault_);
+    badMotorPositionHallSequence0Fault_.hide();
```

Comment:
For all of this stuff, let's encapsulate this to a private function:
`InititalizeLabel(QLabel& label, QLayout& layout)`